section4/exercises/burrito.rb

```diff
@@ -11,9 +11,34 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping(topping)
+    @toppings << topping
+  end
+
+  def remove_topping(to_remove)
```

Comment:
Excellent dynamic methods! Really like seeing you consider your cases with this one- what if a topping is removed that isn't there in the first pllace? Great work!

Reply:
Thanks!